pom.xml

```diff
@@ -144,7 +139,7 @@
             <dependency>
                 <groupId>com.amazonaws</groupId>
                 <artifactId>aws-java-sdk-bom</artifactId>
```

Comment:
We can remove this entire `<dependencyManagement>` block since you added a dependency on the aws-java-sdk plugin.